src/ScrumBoardItBundle/Entity/Issue/AbstractIssue.php

```diff
@@ -187,4 +189,16 @@ public function jsonSerialize()
             'time_box' => $this->getTimeBox()
         );
     }
+
+    public function getNumber()
+    {
+        return $this->number;
+    }
+
+    public function setNumber($number)
+    {
+        $this->number = $number;
+        return $this;
```

Comment:
Symfony convention use to add an empty line before a return, if there is code before